batch/batch/front_end/front_end.py

```diff
@@ -516,16 +516,17 @@ async def _query_batches(request, user):
 WHERE {' AND '.join(where_conditions)}
 GROUP BY batches.id
 ORDER BY batches.id DESC
-LIMIT 50;
+LIMIT 51;
 '''
     sql_args = where_args
 
     batches = [batch_record_to_dict(batch)
                async for batch
                in db.select_and_fetchall(sql, sql_args)]
 
-    if len(batches) == 50:
-        last_batch_id = batches[-1]['id']
+    if len(batches) == 51:
+        last_batch_id = batches[-2]['id']
+        batches = batches[:50]
```

Comment:
This is fine, but I just wanted to let you know that you can drop the last element in Python like this. Your call on what you like better. With the last option, you can write what you're doing a little more clearly:

```
last_batch_id = batches[-2]['id']
batches = batches[:-1:]
```

or

```
batches.pop()
last_batch_id = batches[-1]['id']
```

Reply:
I think I prefer the last option for its clarity.